java/FoodTruckApp2/src/main/java/com/my/FoodTruckApp2/Order/Orders.java

```diff
@@ -0,0 +1,14 @@
+package com.my.FoodTruckApp2.Order;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+
+public class Orders {
+    private Integer id;
+    private Integer customerId;
+}
```

Comment:
This should be singular rather than plural. 1 order has an id and a customer id. Not many orders

Reply:
Ok